File: backend/TrainerPart/TrainerPart/Models/WorkoutSchedule.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace TrainerPart.Models;

public partial class WorkoutSchedule
{
    [Key]
    public int WorkoutId { get; set; }

    public int TrainerId { get; set; }

    public int MemberId { get; set; }

    public string WorkoutDesc { get; set; }

    public int Days { get; set; }

    public virtual Member Member { get; set; } = null!;

    public virtual Trainer Trainer { get; set; } = null!;
}

File: backend/member-service/src/main/java/com/flexify/member/entities/WorkoutSchedule.java
package com.flexify.member.entities;

import jakarta.persistence.*;
import lombok.*;
@Entity
@Table(name = "workout_schedule")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class WorkoutSchedule {
	
	    @Id
	    @GeneratedValue(strategy = GenerationType.IDENTITY)
	    private Integer workoutId;

	    @ManyToOne
	    @JoinColumn(name = "trainer_id", nullable = false)
	    private Trainer trainer;

	    @ManyToOne
	    @JoinColumn(name = "member_id", nullable = false)
	    private Member member;

	    @Column(length = 255)
	    private String workoutDesc;

	    private int days;
	}


